### PR TITLE
{math}[intel/2017b] piSvM-JSC v1.2-20150622

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2707.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3170d6bc673687120c59173f8ab08d8b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2042.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9f25b3b9c956e3a9c43596521b2532a0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2707.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7b7e1a210b6f5b813dd309f1146683b7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2042.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ae4c3160a607c035eecc7d44f5b2b3e1 for a full test report.

File: easybuild/easyconfigs/p/piSvM-JSC/piSvM-JSC-1.2-20150622-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+This version is a fork of the original PiSvM to increase scalability."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/mricherzhagen/pisvm/archive/']
+sources = [{'download_filename': 'b471fcd.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}]
+checksums = ['1ef265888bcf1a64b52c09ead023fd6ed9f01cab32d31ebc84e39b622b15e498']
+
+buildopts = 'CC="$CC" CXXC="$CXX" CFLAGS="$CFLAGS"'
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/piSvM-JSC/piSvM-JSC-1.2-20150622-intel-2017b.eb]: `CXXC` really?

Comment (boegel) [reply easybuild/easyconfigs/p/piSvM-JSC/piSvM-JSC-1.2-20150622-intel-2017b.eb]: Yeah...

**Pull request merged**